src/stores/Form.js

```diff
@@ -1657,12 +1657,11 @@ class FormStore {
         return;
       }
 
-      if(!newValue) {
-        newMetadata[key] = existingValue;
-      }
 
       if(typeof newValue === "object" && typeof existingValue === "object") {
         newMetadata[key] = this.MergeExistingMetadata(existingValue, newValue);
+      } else {
+        newMetadata[key] = newValue;
```

Comment:
@elv-kevin Can you look at this particular change? I know you're trying to preserve existing metadata but this makes it so that any changes that remove a field value don't take effect.

Reply:
Oh yes, i messed up. I think you can just change the first if clause to check for undefined instead of falseish